webui/react/src/e2e/utils/detCLI.ts

```diff
@@ -0,0 +1,34 @@
+import { execSync } from 'child_process';
+import path from 'path';
+
+if (process.env.PW_DET_PATH === undefined) {
```

Comment:
Could we have a default of just `det` or something for running locally for those of us who have `det` on our `$PATH`?

Reply:
it might not work if you have det in a conda environment. I'll make a default, but you'll probably have to use `which det` from inside your conda env to get the path